README.md

```diff
@@ -70,6 +70,25 @@ Wishlist
 | R      | Sep 2      |                   |                                  |
 | S      | Sep 16     |                   |                                  |
 
+### A ladder of models for consideration
+
+It seems useful to start at the bottom of the ladder, both for debugging purposes, but also to get a sense of the added predictive power of each step.
+You get more and more parameters to estimate; how does that balance against the improved accuracy?
+
+1. One human population/geography (e.g. state, HHS region, country), two pathogen populations (dominant variant vs. everything else), binomial dynamics
+   - i.e. `dominant% ~ invlogit[intercept + "slope" * time]`
+3. One geography, multiple variants, multinomial
+4. Multiple geographies, multiple variants, multinomial, no correlations or partial pooling
+   - Statistically equivalent to #2, but requires some different computational implementation
+5. As above plus partial pooling of "slopes" by variant across geographies, but without correlations:
+   - `beta_1ij ~ N(mu_beta1i, sigma_beta1i)`
+   - `mu_beta1i ~ some prior`, `sigma_beta1i ~ some prior`
+   - `i=variant` `j=geography`
+6. As above plus partial pooling of intercepts by variant across geographies
+7. As above plus correlations between "slopes" by variant across counties
+   - i.e. if growth rates of variant A and B are correlated across countries X and Y, then we also expect variants C and D to have correlated growth rates across countries X and Y
```

Comment:
I have often had some trouble conceptualizing this joint space, and what I have settled on is conceptualizing this as about departures from the average. That is, there may be latent unmodeled factors which will make it so that if lineage A is growing faster (than average) in some region, lineage B may as well, while C and D may grow slower.

For example, consider that the variants have tree `((A,B), (C,D))`, and that location X just had a wave of whatever the ancestor of A and B was. Then A+B are likely to grow faster there, and C + D more slowly.